bug: coach validator looks for conventions at src/atdd/ but --export-schemas writes to .atdd/schemas/

## Problem

Two coach validator tests fail in every consumer repo:

```
FAILED test_workflow_consistency.py::test_issue_convention_workflow_includes_smoke
  FileNotFoundError: src/atdd/coach/conventions/issue.convention.yaml

FAILED test_workflow_consistency.py::test_atdd_template_after_coder_points_to_smoke
  FileNotFoundError: src/atdd/coach/templates/ATDD.md
```

**Root cause:** `test_workflow_consistency.py` hardcodes paths relative to `REPO_ROOT`:

```python
# line 18-19
ISSUE_CONVENTION = REPO_ROOT / "src" / "atdd" / "coach" / "conventions" / "issue.convention.yaml"
ATDD_TEMPLATE = REPO_ROOT / "src" / "atdd" / "coach" / "templates" / "ATDD.md"
```

But `atdd init --export-schemas` writes these files to `.atdd/schemas/coach/conventions/`, not `src/atdd/coach/conventions/`.

**Impact:** Blocks `validate-gate` on every PR in consumer repos. Currently blocking janetbusiness/jel-app#246.

## Expected behavior

The validator should resolve convention files from one of:
1. The installed atdd package path (`site-packages/atdd/coach/conventions/`) — these are toolkit files, not consumer repo files
2. Or `.atdd/schemas/` if exported — matching where `--export-schemas` puts them

Not from `src/atdd/` which doesn't exist in consumer repos.

## Fix options

**Option A (preferred):** Validator reads from installed package path. Convention files are part of the toolkit, not the consumer repo. No export needed.

**Option B:** Validator reads from `.atdd/schemas/` and `--export-schemas` is required before validation. Less clean — consumer repos shouldn't need toolkit internals in their tree.

**Option C:** `--export-schemas` writes to `src/atdd/` matching the hardcoded paths. Worst option — pollutes consumer repo with toolkit files.

--- a/src/atdd/coach/validators/test_workflow_consistency.py
+++ b/src/atdd/coach/validators/test_workflow_consistency.py
@@ -12,11 +12,13 @@ from pathlib import Path
 
 import yaml
 
+import atdd
 from atdd.coach.utils.repo import find_repo_root
 
 REPO_ROOT = find_repo_root()
-ISSUE_CONVENTION = REPO_ROOT / "src" / "atdd" / "coach" / "conventions" / "issue.convention.yaml"
-ATDD_TEMPLATE = REPO_ROOT / "src" / "atdd" / "coach" / "templates" / "ATDD.md"
+ATDD_PKG_DIR = Path(atdd.__file__).resolve().parent
+ISSUE_CONVENTION = ATDD_PKG_DIR / "coach" / "conventions" / "issue.convention.yaml"
+ATDD_TEMPLATE = ATDD_PKG_DIR / "coach" / "templates" / "ATDD.md"
 CLAUDE_MD = REPO_ROOT / "CLAUDE.md"
 
 

